features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/CreateEditForm.jsx

```diff
@@ -292,8 +295,15 @@ class CreateEditForm extends React.Component {
         }
     }
     showNameHelper() {
-        const { nameEmpty, nameNotDuplicate } = this.state;
-        if (nameNotDuplicate && !nameEmpty) {
+        const { nameEmpty, nameNotDuplicate, nameMaxLengthExceeds } = this.state;
+        if (nameMaxLengthExceeds) {
+            return (
+                <FormattedMessage
+                    id='Apis.Details.Documents.CreateEditForm.exceeds.document.name.length.helper.text'
+                    defaultMessage='Document name exceeds the max length(60)'
```

Comment:
shall we change this to **Document name exceeds maximum length limit of 60 characters**

Reply:
**Document name exceeds maximum length ~~limit~~ of 60 characters** would be sufficient as we don't need to say limit again since we say maximum length?

Reply:
ya lets cut off limit.

Reply:
Fixed with f4c2bc9